packages/react-aria-components/docs/ComboBox.mdx

```diff
@@ -70,169 +70,130 @@ import {ComboBox, Label, Input, Button, Popover, ListBox, Item} from 'react-aria
 
 <details>
   <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+```css hidden
+@import './Checkbox.mdx';
+@import './ListBox.mdx';
+@import './Popover.mdx';
+```
 
 ```css
```

Comment:
yeah, this one is just a little confusing because it shares styles with another custom wrapper example, and I think it probably shouldn't
maybe I need some way to not export this one, can do in follow up later
for now I've just done the minimal work to get it back to how it used to look